### DISCUSSION TITLE
Models with duplicate ids

### DISCUSSION THREAD

**Discussion by knownasilya** [mikkopaderes/ember-cloud-firestore-adapter#269] [Q&A]
What's the correct way to handle this? I have the same model, but on different parent models. For example:

`account/1/configs/default` and `account/2/configs/default`

I keep running into `The id default has already been used with another 'config' record.`. I've tried `store.unloadRecord(model)` but no luck there it seems.

**Comment by charlesfries** [comment]
I just tried to reproduce this with the latest Ember blueprint, it seems to work for me without that error. The second default record just seems to replace the first one. Take a look at this reproduction https://github.com/charlesfries/adapter-test

**Reply by knownasilya** [reply]
I think my example was wrong, I'm using `createRecord` twice for the same account, basically like an upsert. I should probably just make that an update. 

```js
store.createRecord('config', { id: 'default', ... })
```
